packages/flutter/lib/src/cupertino/text_field.dart

```diff
@@ -365,7 +365,8 @@ class CupertinoTextField extends StatefulWidget {
 
   /// The color to use when painting the cursor.
   ///
-  /// Defaults to the standard iOS blue color. Cannot be null.
+  /// Defaults to activeBlue for light theme, activeOrange for dark theme, or
```

Comment:
please hyperlink to `activeBlue` et al (put the fully qualified name in square brackets)